xml/ay_bigfile.xml

```diff
@@ -1714,10 +1714,39 @@ openssl x509 -noout -fingerprint -sha256
    <sect2 xml:id="CreateProfile.Bootloader.type">
     <title>Loader Type</title>
     <para>
-     Define which boot loader to use: <literal>grub2</literal> or
-     <literal>grub2-efi</literal>.
+     This defines which boot loader (UEFI or BIOS/legacy) to use. Not
+     all architectures support both legacy and EFI variants of the boot
+     loader. The safest (<literal>default</literal>) option is to leave
+     the decision up to the firmware.
```

Comment:
technical details, firmware does not decide. Installer make decision based on set of rules like architecture or if installation is already booted via EFI or legacy. So please change it to "the decision up to the installer."